projects/notebooks/visualizations/rhods-notebooks-performance/analyze/__init__.py

```diff
@@ -1,13 +1,39 @@
-import matrix_benchmarking.common as common
+import json
 import logging
+import numpy as np
+
+from copy import deepcopy
+
+import matrix_benchmarking.common as common
+import matrix_benchmarking.regression as regression
 
 def run():
-    logging.info(f"Received {common.Matrix.count_records()} new entries")
-    for entry in common.Matrix.all_records(): pass
 
     logging.info(f"Received {common.LTS_Matrix.count_records()} historic LTS entries")
-    for lts_entry in common.LTS_Matrix.all_records(): pass
+    lts_entries_raw = list(common.LTS_Matrix.all_records())
+    # Temporarily skip the gathered results
+    lts_entries = list(filter(lambda entry: type(entry.results) is not list, lts_entries_raw))
```

Comment:
can you try to take the 1st value of the gathered entries? 
in theory, all the entries in `entry.results` should contain the same result +/- noise, as they have been executed in the same conditions